### DISCUSSION TITLE
Have issues with Atom Stack A5 pro stopped working after producing banding in laser jobs.

### DISCUSSION THREAD

**Discussion by ADarkGerm** [arkypita/LaserGRBL#1761] [Q&A]
Any advise for me. 
Please & thank you.

Atom Stack A5 pro two months old started to produce banding in images.
I tried a different material, same issue.

Then i tried changing from horizontal to vertical, see if it changed the banding.
The Laser machine then stopped after 10% of job,
Disconnected.

Have not been able to connect since.

Do you think it is dead?

13/02/2022 21:03:11.834	SetStatus   	Machine status [Connecting]
13/02/2022 21:03:11.841	OpenCom     	Open COM4 @ 115200 baud  (UsbSerial)
13/02/2022 21:03:11.907	SendImmediate	Send Immediate Command [0x18]
13/02/2022 21:03:11.918	SetStatus   	Machine status [Disconnected]

How do i install new firmware if it will not connect?
Is the boot loader corrupt?

Their support web site in China is down.
They refuse to answer any emails.

PS Could you also add to your laser GRBL software the ability to set origin with out moving the laser?
I set it in middle of job, then i cannot select it as origin, greyed out until I move it.

Many thanks.

**Comment by arkypita** [comment]
Your log show that machine is not responding to LaserGRBL connection trying.

Assuming you are using the right COM port, the one you can see in windows Device Manager as "USB-SERIAL CH340" and the peripheral driver is not in error (yellow exclamation mark) it should work. If not, maybe re-flashing firmware can solve your issue. You can do it from LaserGRBL tool menu.

![image](https://user-images.githubusercontent.com/8782035/154041845-d8b254ef-18f7-447a-a774-296b48ed3b46.png)


> PS Could you also add to your laser GRBL software the ability to set origin with out moving the laser? I set it in middle of job, then i cannot select it as origin, greyed out until I move it.

If it is grayed out it is because pressing it would have no effect. In fact, if you do not move the laser you would re-assign zero to the very point where you are, and if this is already zero, there would be no variation.

**Reply by ADarkGerm** [reply]
Hi, i finally managed to do a firmware update.

But all eeprom settings were lost for AtomStack A5 Pro
Dont know how the settings got erazed.

Can anyone supply the correct settings to upload, please & thank you.

Tried these settings in configuration 
"
$0=10 (Step pulse time)
$1=255 (Step idle delay)
$2=0 (Step pulse invert)
$3=0 (Step direction invert)
$4=0 (Invert step enable pin)
$5=0 (Invert limit pins)
$6=0 (Invert probe pin)
$10=1 (Status report options)
$11=0.010 (Junction deviation)
$12=0.002 (Arc tolerance)
$13=0 (Report in inches)
$20=0 (Soft limits enable)
$21=0 (Hard limits enable)
$22=0 (Homing cycle enable)
$23=0 (Homing direction invert)
$24=25.000 (Homing locate feed rate)
$25=500.000 (Homing search seek rate)
$26=250 (Homing switch debounce delay)
$27=1.000 (Homing switch pull-off distance)
$30=255 (Maximum spindle speed)
$31=0 (Minimum spindle speed)
$32=0 (Laser-mode enable)
$100=250.000 (X-axis travel resolution)
$101=250.000 (Y-axis travel resolution)
$102=0.000 (Z-axis travel resolution)
$110=500.000 (X-axis maximum rate)
$111=500.000 (Y-axis maximum rate)
$112=0.000 (Z-axis maximum rate)
$120=10.000 (X-axis acceleration)
$121=10.000 (Y-axis acceleration)
$122=0.000 (Z-axis acceleration)
$130=200.000 (X-axis maximum travel)
$131=200.000 (Y-axis maximum travel)
$132=0.000 (Z-axis maximum travel)
"

I must have someting wrong in this setup or the hardware is bust..
AtomStack will not provide support. 
Poor company to get help with, in my opinion.

But the machine does not work.
Laser comes on when requested at least, but no movement at all.
Tried 1.1f & 1.1h both dont work.

Also LaserGRBL configuration say export settings to NC file, should this not be named another name?
Bit confusing.

Also how do you use the materials database in jobs and how do you select AtomStack A5 pro as machine?

Please help & Thanks for advice.

**Reply by arkypita** [reply]
Try asking here [https://www.facebook.com/groups/atomstack3d](https://www.facebook.com/groups/atomstack3d) or here [https://www.facebook.com/groups/497474664689694](https://www.facebook.com/groups/497474664689694)

I am sure you can find someone with same machine that can share his settings with you.
